docs/supported-data-types-in-carbondata.md

```diff
@@ -36,8 +36,6 @@
 
   * String Types
     * STRING
-    * CHAR
-    * VARCHAR
```

Comment:
Still, create table (a varchar(20), b char(20) stored as carbondata, are created as STRING type in carbondata with SPARK 3.1. [CARBONDATA-4226](https://issues.apache.org/jira/browse/CARBONDATA-4226) is not yet resolved. Can Refer org.apache.spark.sql.parser.CarbonSpark2SqlParser#getFields

Reply:
As discussed in dev community ,  CARBONDATA-4226 be resolved.